lib/Logger/Logger.cpp

```diff
@@ -704,64 +714,62 @@ void Logger::append(LogGroup& group,
 ////////////////////////////////////////////////////////////////////////////////
 
 void Logger::initialize(application_features::ApplicationServer& server, bool threaded) {
-  MUTEX_LOCKER(locker, _initializeMutex);
-
-  if (_active.exchange(true)) {
+  if (_active.exchange(true, std::memory_order_acquire)) {
     THROW_ARANGO_EXCEPTION_MESSAGE(TRI_ERROR_INTERNAL,
                                    "Logger already initialized");
   }
 
   // logging is now active
-  TRI_ASSERT(_active);
-  
   if (threaded) {
-    _loggingThread = std::make_unique<LogThread>(server, ::LogThreadName);
-    if (!_loggingThread->start()) {
+    auto loggingThread = new LogThread(server, ::LogThreadName);
```

Comment:
Can we make it a unique_ptr again? Otherwise we may leak it if `loggingThread->start()` fails. It's not highly important because the program will abort then anyway, but just out of habit we should not use raw `new` ops if avoidable.
```suggestion
    auto loggingThread = std::make_unique<LogThread>(server, ::LogThreadName);
```